airflow/ui/src/components/TriggerDag/TriggerDAGModal.tsx

```diff
@@ -43,21 +42,14 @@ const TriggerDAGModal: React.FC<TriggerDAGModalProps> = ({
     <Dialog.Content backdrop>
       <Dialog.Header paddingBottom={0}>
         <VStack align="start" gap={4}>
-          <Heading size="xl">
-            Trigger Dag - {dagDisplayName} <TogglePause dagId={dagId} isPaused={isPaused} skipConfirm />
```

Comment:
Oh, a bit sad - I was actually happy about the toggle pause button on the top. The one on the bottom is also working...

Reply:
> Oh, a bit sad - I was actually happy about the toggle pause button on the top.

I got quite a bit of user feedback that people were confused with the change and that it forced extra clicks.

> The one on the bottom is also working...

What do you mean?

Reply:
> > The one on the bottom is also working...
> 
> What do you mean?

Wanted to say: "Also OK" / "also acceptable"